easybuild/easyconfigs/m/ml_dtypes/ml_dtypes-0.3.2_EigenAvx512.patch

```diff
@@ -0,0 +1,1217 @@
+# ml_dtype 0.3.2 provides Eigen commit 7bf2968. src/Core/arch/AVX512/TrsmUnrolls.inc is missing.
```

Comment:
@ThomasHoffmann77 Some more info should be provided here...

Which copy of `TrsmUnrolls.inc` is being injected through this patch, which version/commit?

Reply:
@boegel I updated the comment in the patch.